prep-array-challenge/src/compact.js

```diff
@@ -0,0 +1,12 @@
+/* exported compact */
+function compact(array) {
+  const compactArray = [];
+  for (const element of array) {
+    //console.log(element);
+    if (Boolean(element)) {
+      compactArray.push(element);
+    }
```

Comment:
You actually don't need to use `Boolean` here. You can just do `if (element)`. `if` statements will always evaluate to `true` or `false`, so by doing `if (element)`, we will check to see if that variable evaluates to `true` or `false`, which will get us the result we want!